components/CameraView.tsx

```diff
@@ -38,6 +61,8 @@ const CameraView = (): JSX.Element => {
 				type={Camera.Constants.Type.back}
 			/>
 			<PhotoButton onPress={onPressPhotoButton} color="#f4e1e6" />
+			<Text>{lastImageURI}</Text>
+			<Text>{status}</Text>
```

Comment:
```suggestion
			<Text>{lastImageURI}</Text>
			<Text>{status}</Text>
```

can we mark these as temporary somehow so we know to get rid of them later?

Reply:
added a comment